From 6ea35989f448ac416e1681ef224e829edc6ab4df Mon Sep 17 00:00:00 2001
From: Dan Poltawski <dan@moodle.com>
Date: Fri, 19 Aug 2016 13:00:18 +0100
Subject: [PATCH 1/1] MDL-12345 amos: dummy commit

Here we are going to test the commit checker..

This commit should fail because i'm introducing an amos syntax error
with nametextarea and mod__bar

AMOS BEGIN
 MOV [nametextarea, mod_data][fieldtypelabel, datafield_textarea]
 CPY [foo,bar],[foo,mod__bar]
AMOS END
---
 mod/data/lang/en/data.php | 1 -
 1 file changed, 1 deletion(-)

diff --git a/mod/data/lang/en/data.php b/mod/data/lang/en/data.php
index 0d01c13..e4bb55c 100644
--- a/mod/data/lang/en/data.php
+++ b/mod/data/lang/en/data.php
@@ -259,7 +259,6 @@ $string['namenumber'] = 'Number field';
 $string['namepicture'] = 'Picture field';
 $string['nameradiobutton'] = 'Radio button field';
 $string['nametext'] = 'Text field';
-$string['nametextarea'] = 'Textarea field';
 $string['nameurl'] = 'URL field';
 $string['newentry'] = 'New entry';
 $string['newfield'] = 'Create a new field';
-- 
2.9.0

